glob/braces.in

```diff
@@ -6,6 +6,10 @@ root=true
 [*.{py,js,html}]
 choice=true
 
+; word choice with spaces
+[*.{py, js, html}]
+choice=true
```

Comment:
Please change this property's value.  E.g., 
```suggestion
choice=spaces
```
That way the new tests will detect that this block is being applied.  If this block and the previous block have the same value, a passing test doesn't actually confirm that this block was applied.

Reply:
I renamed the property value